tools/internal_ci/linux/grpc_xds_k8s_run_xtest.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## xDS test server/client Docker images
+readonly IMAGE_REPO="gcr.io/grpc-testing/xds-interop"
+
+find_latest() {
+  gcloud container images list-tags --filter=tags\~v1\.\\d\\d\.x "${IMAGE_REPO}/${1}-${2}" '--format=table[no-heading](tags)' | awk -F, '{print $2}' | sort | tail -n 1
+}
+
+if [ "${LATEST_BRANCH}" == "" ]; then
+  cpp_server=$(find_latest cpp server)
```

Comment:
Do we  need to check clients and servers? I think servers should be enough. I can't imagine a situation where the server exists, and the client doesn't (for the main languages). If this happens, I'd treat it as an error that needs to be reported and fixed.

Reply:
I would leave the client check there. For a new release (branch) if the client image doesn't get created (for whatever reason) then this will "fall back" to the latest for which both images exist. I hope the error of a client (or server) image not getting created will be caught in a different place or process.